src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -102,7 +103,10 @@ public static FlutterSdk getIncomplete(@NotNull final Project project) {
 
   @Nullable
   public static FlutterSdk forPath(@NotNull final String path) {
-    final VirtualFile home = LocalFileSystem.getInstance().findFileByPath(path);
+    VirtualFile home = LocalFileSystem.getInstance().findFileByPath(path);
+    if (home == null) {
+      home = LocalFileSystem.getInstance().refreshAndFindFileByIoFile(new File(path));
```

Comment:
This call worries me as I recall issues where it blocked the UI thread.  Having said that I'm racking my brain for more context.  Does this ring any bells?

Reply:
How's that for an unhelpful comment! 😉

Reply:
Good point. This could be called from anywhere (I guess). I'll move the refresh code into the wizard.